travis.sbt

```diff
@@ -0,0 +1,32 @@
+commands ++= Seq(
+  // check policies, if on master also upload
+  Command.command("whitesourceCi") { state =>
+    val extracted = Project.extract(state)
+    import extracted._
+
+    if (sys.env.getOrElse("TRAVIS_SECURE_ENV_VARS", "") == "true") {
+      val stateWithCredentials =
+        append(credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY")), state)
+
+      runTask(whitesourceCheckPolicies, stateWithCredentials)
+
+      if (sys.env.getOrElse("TRAVIS_BRANCH", "") == "master" && sys.env.getOrElse("TRAVIS_EVENT_TYPE", "") == "push") {
+        runTask(whitesourceUpdate, stateWithCredentials)
```

Comment:
Shouldn't that run only on releases?

Reply:
I am not sure what the policy is. @raboof?